### ISSUE TITLE
Saving a multipart/form request sets the type to x-www-form-url-encoded and removes the content-type header.

### ISSUE THREAD

**Issue by jarrodek** [jarrodek/ChromeRestClient#246]
Originally reported on Google Code with ID 250
```
What steps will reproduce the problem?
1. Create a multipart/form-data request.
2. Add any form value, or any file.
3. Click the save button to save the request for later.
4. The request has now changed type, lost a content-type header, and removed any files
from the request.

What is the expected output? What do you see instead?
It should be the same type as before (multipart/form-data), it is now x-www-form-urlencoded.
It should preserve the boundary string (content-tpye header), it does not. The files
that were previously selected for upload, are no longer selected.

On what operating system, browser and browser version?
Crunchbang linux: kernel version 3.12.7 . Advanced rest client 3.1.7 on Chromium version
31.0.1650.63 Built on Debian 7.2, running on Debian 7.3

Please provide any additional information below.


```

Reported by `NotCole` on 2014-04-23 18:51:33

**Comment by jarrodek** [comment]
This may not be an issue. Right now there is Content-Type dropdown which may confuse the user about it's purpose. Meanwhile if the content type header is one of the defined in the dropdown it will be removed from headers section.
This behaviour is not natural and cause confusion. This dropdown will be removed in the future.

As for a file, you may want to see #250 for more information.

**Comment by jarrodek** [comment]
It's fixed in v4.5.0
